VTUL/iawa
# LIBTD-1845: batch import collection metadata

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1845) (:star:)

# What does this Pull Request do? (:star:)
This PR imports a batch of collection metadata at once.

# What's the changes? (:star:)

* Auto insert title into bibliographic citation metadata
* Add batch_collection_import task

# How should this be tested?

* Run as railsapps: RAILS_ENV=production bundle exec rake iawa:batch_collection_import["collection_metadata.csv"] 
* See 11 collections have corresponding metadata information

# Interested parties
Tag (@pmather ) interested parties

(:star:) Required fields
